index.html

```diff
@@ -1684,27 +1684,44 @@ <h3>
           takes a <a>USVString</a> <var>theme color</var> as an argument. This
           algorithm returns a <code>USVString?</code>.
         </p>
-        <div class="issue" data-number="760"></div>
         <ol>
-          <li>Let <var>potential color</var> be the result of running
-          [[CSS-SYNTAX-3]]'s <a>parse a component value</a> algorithm with
-          <var>theme color</var> as input. If parsing returns a syntax error,
-          return <code>undefined</code>.
-          </li>
-          <li>Let <var>color</var> be the result of attempting to parse
-          <var>potential color</var> as a CSS color, as per [[CSS-SYNTAX-3]].
-          If parsing fails:
+          <li>Let |color| be the result of [=CSS/parsing=] |theme color| as a
+          CSS <a>&lt;color&gt;</a>. [[CSS-SYNTAX-3]]
```

Comment:
Note, "CSS/parsing" and "&lt;color&gt;" are not things in the exported terms database - at least that we can find at the moment via https://respec.org/xref... @sidvishnoi can check why they are not there.  

@christianliebel, when there is a linking error, ReSpec tries to tell you how to fix it at https://respec.org/xref. I can't see `sRGB` at https://respec.org/xref either.

Reply:
Found it: [=CSS/parse=] should work... 

```suggestion
          <li>Let |color| be the result of [=CSS/parse=] |theme color| as a
          CSS &lt;color&gt;.
```

Reply:
`[=CSS/parsing=]` works, but need to add an alias for css-parsing in `respecConfig.localBiblio` until https://github.com/tobie/specref/pull/564 is merged.
``` js
localBiblio: {
  "css-parsing": { aliasOf: "css-parsing-3" },
}
```

Reply:
> Found it: [=CSS/parse=] should work...

Note that Bikeshed automatically handles common conjugations of the term, so you can write things like `[=CSS/parsing=]` and have it automatically resolve to `[=CSS/parse=]`; perhaps ReSpec should do the same?

Reply:
@tabatkins `[=CSS/parsing=]` also works in ReSpec, the problem was with Specref API
See https://github.com/w3c/manifest/pull/878#discussion_r432331678